packages/web/src/components/basic/SelectedFilters.js

```diff
@@ -144,7 +164,7 @@ SelectedFilters.propTypes = {
 	className: types.string,
 	clearAllLabel: types.title,
 	innerClass: types.style,
-	showClearAll: types.bool,
+	showClearAll: types.showClearAll,
```

Comment:
@ShahAnuj2610 What is the default value of it?

Reply:
true

Reply:
https://github.com/appbaseio/reactivesearch/blob/08b63715172c8c29c194d158b66d2302762d9b24/packages/web/src/components/basic/SelectedFilters.js#L180